gdpr/README.md

```diff
@@ -0,0 +1,111 @@
+# GDPR and Data Privacy Guide
+
+> Parts of this section are based on an article, 
+[A Developer’s Guide to GDPR that won’t make you sweat](https://medium.com/ft-product-technology/a-developers-guide-to-gdpr-that-won-t-make-you-sweat-4f1f7f1d9c8b) by [Mags Allen
+](https://medium.com/@mags_40745).
+
+> More reading on how we handle GDPR as Django 
+Girls Foundation can be found on our website's
+ [Terms and Conditions](https://djangogirls.org/terms-conditions/)
+ and [Cookie and Privacy Policy](https://djangogirls.org/privacy-cookies/) 
+ sections.
+
+**The General Data Protection Regulation (GDPR) is a
+ set of rules that came into effect on May 25th, 2018.** 
+ 
+It regulates the use of data by organisations.
+It applies to any organisation that stores or processes
+ any data relating to individuals inside the European 
+ Union (EU). 
+ 
+ Since we have events in Europe, it is important 
+ for Django Girls Foundation and organizers across 
+ the globe to comply with GDPR as it has legal 
+ implications. We therefore recommend all organizers 
+ to comply with GDPR as it protects users' data, 
+ irrespective of their location. 
+ 
+ This guide therefore applies to all organizers 
+ as they deal with the sensitive data of fellow 
+ organizers, attendees and sometimes, sponsors 
+ from the EU region. Every event organizer should 
+ ensure their event is GDPR compliant to avoid 
+ chances of being sued by either applicants or
+ sponsors.
+ 
+# Storage of Data
+
+> Protect the data you store and only keep what you need.
+
+When you customise the application, ensure that you 
+only ask for data that you need to run your event 
+successfully. This data should be protected and 
+not shared with people who do not participate in 
+ensuring the successful delivery of the workshop. 
+For example, coaches should not be given email 
+addresses or phone numbers of attendees who are not in their group
+as they do not need to contact them before or 
+after the workshop.
+
+Coaches and organizers need to be informed that 
+they should protect personal data of attendees and
+other coaches/organizers and not share it amongst
+themselves without the consent of the person 
+involved.
+
+# Subject Access Requests
+
+> Make it easy to retrieve, delete and anonymise 
+data when you need to.
```

Comment:
Should we add the section here?

Reply:
I have added the "OK with Photo" or "Not OK with photo here" as many organizers seem to just publish photos without asking for consent, under the anonymise data so maybe we can still add that here. What do you think?